airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/source/relationaldb/InitialLoadTimeoutUtil.kt

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.integrations.source.relationaldb
+
+import com.fasterxml.jackson.databind.JsonNode
+import io.github.oshai.kotlinlogging.KotlinLogging
+import java.time.Duration
+import java.util.*
+
+private val LOGGER = KotlinLogging.logger {}
+
+object InitialLoadTimeoutUtil {
+
+    val MIN_INITIAL_LOAD_TIMEOUT: Duration = Duration.ofHours(4)
+    val MAX_INITIAL_LOAD_TIMEOUT: Duration = Duration.ofHours(24)
```

Comment:
nit: This _feels_ like the kind of thing some large enterprise customer is going to want to change one day...

Reply:
Ack - we can always change these limits as we discover those use cases

Reply:
I think the one day upper limit is good enough for now and actually would want to prevent customers from setting it to higher for day. We have ~20 attempts now, so this still gives 20 days for Airbyte to finish a customer's sync. Let me know if we should revisit these limits in the first iteration

Reply:
It's probably fine for now!